spec/features/locales_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'locale' do
```

Comment:
You're not describing `locale`-  you're describing `locale switching` or something like that.

Pay attention to these